gpfit/fit.py

```diff
@@ -202,5 +202,5 @@ def rfun(p):
         if K == 1:
             cstrt = MonomialEquality(mono2, "=", mono1)
 
-    return cstrt, rmsErr
```

Comment:
I agree that `params` is a bit scary - GPfit needs to be refactored and this should be the first thing on the chopping block - but given that `print_fit` and `plot_fit` both directly use `params`, it seems right to return it.